app/controllers/nonsipp/ViewOnlyTaskListController.scala

```diff
@@ -632,7 +636,7 @@ object ViewOnlyTaskListController {
     val quotedSharesStatusAndLink: TaskListStatus = getCompletedOrUpdatedTaskListStatus(
       currentUA,
       previousUA,
-      pages.nonsipp.totalvaluequotedshares.Paths.quotedShares
+      pages.nonsipp.totalvaluequotedshares.Paths.quotedShares \ "totalValueQuotedShares"
```

Comment:
This is necessary based on the way our implementation works around there being no field in ETMP for the Yes/No page answer (`quotedSharesManagedFundsHeld`). Because of this, we don't want that field included in this comparison, we only want to compare this monetary value.